tests/e2e/models/BUILD

```diff
@@ -80,7 +80,11 @@ iree_check_single_backend_test_suite(
     name = "check_cuda_cuda",
     timeout = "long",
     srcs = CHECK_FRAMEWORK_TESTS,
-    compiler_flags = ["--iree-input-type=mhlo"],
+    compiler_flags = [
+        "--iree-input-type=mhlo",
+        #TODO(#12445): Drop this flag once the test is fixed.
+        "--iree-codegen-llvmgpu-enable-transform-dialect-jit=false",
```

Comment:
If a flag is broken (indicated by one test here - and we are really short on test coverage for whole-program compilation), shouldn't we disable the flag / flip it to default-off, instead of disabling or tweaking a test?

Reply:
That's what I'm wondering: if a customer comes in and tries to compile mobilenetv3 with default flags, what happens?

Reply:
My impression is that it's not clear what's really broken at this point, i.e., which pattern or other part of the infra is the root cause. @nicolasvasilache recommended disabling this flag as a workaround, but I'm not sure about disabling it globally.

Reply:
Sorry to be the grumpy one, but this is a core model, and if it is broken with default flags, we need to pursue that with some priority.

Reply:
BTW, compiler priorities:

1. don't crash
2. don't error
3. generate correct code
4. generate efficient code

the error from the issue is 2. If a code path is producing errors for real programs, that is higher priority to fix than (for example) benchmark regressions. We should be _increasing_ test coverage for new features so we have enough confidence in 1-3 before over-indexing on 4.

Reply:
I haven't heard back from from @nicolasvasilache on this, so I decided to change the default globally and see if the other tests pass.